### ISSUE TITLE
SQL Error on Blog Export

### ISSUE THREAD

**Issue by nozreh** [rainlab/blog-plugin#477]
I was trying to export our existing blogs but I'm having issue right now 

Octobercms Build: 458
Blog Plugin version: 1.3.4
Database: Postgres 9.6.12

``"SQLSTATE[42703]: Undefined column: 7 ERROR: column "featured_images" does not exist LINE 1: select * from "system_files" where field = "featured_images"... ^ (SQL: select * from "system_files" where field = "featured_images" AND attachment_type = "RainLab\\Blog\\Models\\Post" and "system_files"."attachment_id" in (4, 6, 7, 8, 9, 10,) order by "sort_order" asc)" on line 664 of /vendor/laravel/framework/src/Illuminate/Database/Connection.php``

**Comment by w20k** [comment]
Hm...worked for me, but without featured image link. Will try it under the Postgres as well.

Dummy example:
```
ID,Title,Content,"HTML Content",Excerpt,Slug,Categories,"Author Email","Featured Images","Created date","Updated date","Published date"
1,"First blog post","This is your first ever **blog post**! It might be a good idea to update this post with some more relevant content.

You can edit this content by selecting **Blog** from the administration back-end menu.

*Enjoy the good times!*","<p>This is your first ever <strong>blog post</strong>! It might be a good idea to update this post with some more relevant content.</p>
<p>You can edit this content by selecting <strong>Blog</strong> from the administration back-end menu.</p>
<p><em>Enjoy the good times!</em></p>","The first ever blog post is here. It might be a good idea to update this post with some more relevant content.",first-blog-post,Uncategorized,admin@domain.tld,,"2019-10-10 02:11:08","2019-10-10 02:11:44","2019-10-10 02:11:08"
2,erewrew,vxcvxcvzxcvdsfdsfdsaf,<p>vxcvxcvzxcvdsfdsfdsaf</p>,sdfsdfsdf,erewrew,Uncategorized,admin@domain.tld,,"2019-10-10 02:14:07","2019-10-10 02:14:07","2019-10-11 00:00:00"
```

**Comment by bennothommo** [comment]
It's a confirmed bug - PostgreSQL treats lower-case strings inside quotes as column names.

**Comment by bennothommo** [comment]
Fixed by https://github.com/rainlab/blog-plugin/commit/f4b16deabd9181b6ff46e842dae961be2426ea90 - will be released with version 1.3.5 of the Blog plugin.
